build/tools/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM alpine:3.9
```

Comment:
maybe we can avoid creating a Docker image for generating certs ? This could be done manually or as a step of the cloud Dockerfile ?